spec/models/application_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Application, type: :model do
+  it { should belong_to :user }
+end
```

Comment:
Great to see your test-driven workflow involves writing feature specs and lower-level specs (e.g. model specs) in the same PR branch. Keep it up! :+1: